### ISSUE TITLE
Fix overheal

### ISSUE THREAD

**Issue by atillagallio** [telemmo/telemmo#83]

**Comment by PlatypusKing** [comment]
Personally I think due to the random nature of skills it's better to leave overheal in.  Damage is never wasted, but if you cast a heal your first turn of a fight then that will do nothing without overheal.  This would favor damage skills drastically in the balance.

**Comment by KurzedMetal** [comment]
I agree with @PlatypusKing, I never saw overheal as a bug, but a feature for healers.

If this change happens to get implemented, heal should get buffed because of the case that @PlatypusKing described.
